### PR TITLE
Fixes Issue #28

### PR DESCRIPTION
**Pull request by kaitlyn-zeichick**
Atlanta was swapped out for Charleston because Atlanta already exists in the dataset. Charleston has the closest latitude and jan temp to Atlanta and isn't in the dataset.

### CODE REVIEW

File: ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst [hunk 1]
```diff
@@ -6,22 +6,28 @@
...
-Atlanta, Georgia, which has a latitude of 33 degrees, is not included in
-this weather dataset. If you were asked to predict average January
+Charleston, South Carolina, which has a latitude of 33 degrees, is not included 
+in this weather dataset. If you were asked to predict average January
 temperatures there, you might make a good guess and say 52 degrees F,
 because that is the mean temperature for southern cities with a latitude
-below 35. The problem with this prediction is that it is the same for
+below 35. This was found using the bar chart that you saw in the introduction:
+
+.. image:: figures/scatter3.png
```
Comment (raskutti) [inline ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: This image isn't added in this PR, where is it?

**Review by raskutti — CHANGES_REQUESTED** [review]
Generally looks good. Please comment in the PR description why you are swapping out Atlanta for Charleston.

Also, make sure the CLA is good, and remove pavement.py from this PR.

File: ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst (hunk 1 above)
Comment (kaitlyn-zeichick) [reply ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: scatter3.png already existed. It was used in a different section, and I didn't take it out so it should still be in figures.

**Comment by lindseyc** [comment]
@googlebot I consent.

File: ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst (hunk 1 above)
Comment (raskutti) [reply ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: Ok LGTM. Once pavement.py is removed from the PR, we can merge.

Comment (raskutti) [reply ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: @kaitlyn-zeichick

Comment (raskutti) [reply ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: Reminder to get this fixed @kaitlyn-zeichick @lindseyc

Comment (kaitlyn-zeichick) [reply ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: Pavement.py was removed. Did I do it incorrectly?

**Comment by raskutti** [comment]
I think you deleted the file completely, whereas we want to remove it from
the PR. So, it should be returned to its state in master.

File: ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst (hunk 1 above)
Comment (kaitlyn-zeichick) [reply ac1/_sources/scatter_plots_and_correlation/scatter_plots.rst:15]: Just added it back!

**Review by raskutti — APPROVED** [review]

**Pull request merged**